UI.tsv

```diff
@@ -119,15 +119,15 @@ UI_20150317_000118	{@st42b}Move map{/}
 UI_20150317_000119	{@st42b}Combat Order{/}
 UI_20150317_000120	{@st41b}Close{/}
 UI_20150317_000121	{s16}{b}Create items by clicking on the Create button or double-click.
-UI_20150317_000122	{s16}{b}ea
+UI_20150317_000122	{s16}{b} articles
```

Comment:
It is ea as in each. But it is still not known how to phrase this differently. 'articles' is incorrect.